content/en/news/2019/announcing-1.4/change-notes/index.md

```diff
@@ -16,6 +16,12 @@ weight: 10
 
 ## Telemetry
 
+- **Added** experimental [in-proxy](/docs/ops/telemetry/in-proxy-service-telemetry/) telemetry reporting to Stackdriver.
+- **Added** experimental support for generating TCP service metrics [in-proxy](/docs/ops/telemetry/in-proxy-service-telemetry/).
+- **Improved** support for [in-proxy](/docs/ops/telemetry/in-proxy-service-telemetry/) Prometheus generation of HTTP service metrics (from experimental to alpha).
+- **Improved** stability for proxy metadata exchange (from experimental to alpha).
```

Comment:
@kyessenov is this a user facing change? My understand is this is more of an implementation detail of in-proxy metric generation?

Reply:
It's not user facing. I think we want one item for improved stability of in-proxy runtime (Wasm and our filters).